app/controllers/V1/nsm_claims_controller.rb

```diff
@@ -0,0 +1,27 @@
+module V1
+  class NsmClaimsController < ApplicationController
+    def update
+      current_nsm_claim.update!(allowed_params)
+      render json: current_nsm_claim, status: :created
+    rescue ActiveRecord::RecordInvalid => e
+      report_error(e)
+      render json: { errors: e.message }, status: :unprocessable_entity
+    rescue ActiveRecord::RecordNotFound
+      report_error(e)
+      head :not_found
+    end
+
+  private
+
+    def allowed_params
```

Comment:
could this be permitted_params in line with the assigned_counsel_claims patch method?